src/main/scala/temple/generate/service/go/GoServiceGenerator.scala

```diff
@@ -212,4 +212,12 @@ object GoServiceGenerator extends ServiceGenerator {
       ),
     )).map { case (path, contents) => path -> (contents + "\n") }
   }
+
+  private def verb(endpoint: Endpoint): String = endpoint match {
+    case Endpoint.ReadAll => "List"
+    case Endpoint.Create  => "Create"
+    case Endpoint.Read    => "Read"
+    case Endpoint.Update  => "Update"
+    case Endpoint.Delete  => "Delete"
+  }
```

Comment:
should this instead be placed in `ServiceGenerator`? Is it Go-specific?

Reply:
🚚 -> Endpoint

Reply:
Don't think so, happy for it to be moved